apps/frontend/src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
+import { Navigate } from "react-router-dom";
```

Comment:
let's re-export it from common components
```suggestion
import { Navigate } from "src/libs/components/components";
```

Reply:
Should I add it there?

Reply:
yes, let's re-export it from common components, the same as we do for NavLink